docs/design/autoware-architecture/index.md

```diff
@@ -0,0 +1,41 @@
+# Architecture Overview
+
+## Introduction
```

Comment:
[Nits] I think you can write this sentence here.

```suggestion
## Introduction

This page describes the architecture of Autoware.
```

Reply:
As a general comment, I think that if a page has a title of X, then there is no need to have a sentence at the start like "This page describes X". It should be clear from the title and the section of the documentation you are in, as shown by the left sidebar.

Reply:
> there is no need to have a sentence at the start like "This page describes X".

Exactly. I'd like to add a bit more additional information here to explain the scope and differences from other pages.

Reply:
fixed in 81d4bdcde01a5